operator/clusterutilities.go

```diff
@@ -683,96 +682,51 @@ func GetPgmonitorEnvVars(metricsEnabled string) string {
 func GetPgbackrestS3EnvVars(cluster crv1.Pgcluster, clientset *kubernetes.Clientset,
 	ns string) string {
 
-	if cluster.Labels[config.LABEL_BACKREST] == "true" &&
-		strings.Contains(cluster.Spec.UserLabels[config.LABEL_BACKREST_STORAGE_TYPE], "s3") {
-
-		// populate the S3 bucket, endpoint and region using either the values in the pgcluster
-		// spec (if present), otherwise populate using the values from the pgo.yaml config file
-		s3EnvVars := PgbackrestS3EnvVarsTemplateFields{}
-		if cluster.Spec.BackrestS3Bucket != "" {
-			s3EnvVars.PgbackrestS3Bucket = cluster.Spec.BackrestS3Bucket
-		} else {
-			s3EnvVars.PgbackrestS3Bucket = Pgo.Cluster.BackrestS3Bucket
-		}
-
-		if cluster.Spec.BackrestS3Endpoint != "" {
-			s3EnvVars.PgbackrestS3Endpoint = cluster.Spec.BackrestS3Endpoint
-		} else {
-			s3EnvVars.PgbackrestS3Endpoint = Pgo.Cluster.BackrestS3Endpoint
-		}
-
-		if cluster.Spec.BackrestS3Region != "" {
-			s3EnvVars.PgbackrestS3Region = cluster.Spec.BackrestS3Region
-		} else {
-			s3EnvVars.PgbackrestS3Region = Pgo.Cluster.BackrestS3Region
-		}
+	if !(cluster.Labels[config.LABEL_BACKREST] == "true" &&
```

Comment:
I realize this is just carried over from the previous code, but at this point we really don't need the check of the backrest label since backrest is essentially always enabled as of 4.2.  And since this is one of those labels that needs to ultimately be removed holistically across the entire codebase, I'm def fine with leaving until we do so.